style-sheets/landing-page-components-overrides.scss

```diff
@@ -0,0 +1,163 @@
+@use '../design-tokens/colors.scss' as colors;
+
+/* ------------------------------- Landing Page ------------------------------- */
+
+.row {
+    margin-left: auto !important;
+    padding-left: 1rem !important;
+}
+
+.logo {
+    margin-top: 2.5rem;
+}
+.full-width {
+    max-width: 100%;
+    overflow: hidden;
+}
+.landing-grid {
+    padding-right: 0;
+    height: 100vh;
+}
+
+.landing-title {
+    margin-top: 4.375rem;
+    font-size: 4.75rem;
+    font-weight: 400;
+    line-height: 5.375rem;
+}
+
+.landing-subtitle {
+    margin-top: 0.5rem;
+    color: colors.$blue-60;
+    font-size: 2rem;
+    font-weight: 400;
+    line-height: 2.5rem;
+}
+
+.landing-desc {
+    margin-top: 2.5rem;
+    color: colors.$gray-70;
+    font-size: 2rem;
+    font-weight: 400;
+    line-height: 2.5rem;
+    margin-bottom: 3.75rem;
+}
+
+.btn-landing {
```

Comment:
Hi Nick, let's rename this, "btn" is the bootstrap button name, let's just call it "button-landing" or something else

Reply:
Thanks, I will change it.